pin-project-internal/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "pin-project-internal"
+# NB: When modifying, also modify html_root_url in lib.rs
+version = "0.3.4"
+authors = ["Taiki Endo <te316e89@gmail.com>"]
+edition = "2018"
+license = "Apache-2.0/MIT"
+description = "An interal crate to support pin_project - do not use directly"
+repository = "https://github.com/taiki-e/pin-project"
+documentation = "https://docs.rs/pin-project/"
+readme = "README.md"
+keywords = ["pin", "macros", "attribute"]
+categories = ["rust-patterns"]
+exclude = ["/.travis.yml", "/bors.toml"]
+
+[lib]
+proc-macro = true
+
+[features]
+# Default features.
+default = ["project_attr"]
+# Enable to use `project` attribute.
+project_attr = ["syn/visit-mut"]
+# Enable to allow using the crate with a renamed 'pin-project' dependency
+renamed = ["proc-macro-crate"]
```

Comment:
Could you make `serde` and `lazy_static` optional and add to this?

Reply:
Done

Reply:
Thanks